test_extraction_DK.py

```diff
@@ -4,12 +4,88 @@
 import time
 from extract import extract_to_list
 from torch.utils.data import DataLoader
+from typing import Optional
 
```

Comment:
add default model dir
```suggestion
DEFAULT_MODEL_DIR = Path(Path.home(), ".relation_model")
```